tests/load/locustfiles/load.py

```diff
@@ -85,3 +86,11 @@ def tick(self) -> TickTuple | None:
         spawn_rate: float = max(abs(users - self.get_current_user_count()), 1)
 
         return users, spawn_rate, self.user_classes
+
+
+class StoredNotifAutopushLoadTestShape(AutopushLoadTestShape):
```

Comment:
Adding `StoredNotifAutopushUser` to `AutopushLoadTestShape` seemed to force both classes usage during the load test. So I setup this subclass for now so we can focus on solely `StoredNotifAutopushUser`, we can revisit this later.